x-pack/plugins/observability_solution/observability_ai_assistant_app/public/utils/storybook_decorator.tsx

```diff
@@ -22,20 +23,45 @@ const mockChatService: ObservabilityAIAssistantChatService = createStorybookChat
 
 export function KibanaReactStorybookDecorator(Story: ComponentType) {
   const ObservabilityAIAssistantChatServiceContext = React.createContext(mockChatService);
+  const ObservabilityAIAssistantMultipaneFlyoutContext = React.createContext({
+    container: <div />,
+    setVisibility: () => false,
+  });
+
   return (
     <KibanaContextProvider
       services={{
-        triggersActionsUi: { getAddRuleFlyout: {} },
+        application: {
+          navigateToApp: () => {},
+        },
+        http: {
+          basePath: {
+            prepend: () => '',
+          },
+        },
+        licensing: {
+          license$: new Subject(),
+        },
```

Comment:
can we use the mock contract from core (and other plugins if they have it available)? We can make jest work by adding the following lines to `preview.js`:

```ts
import * as jest from 'jest-mock';

window.jest = jest;
```